packages/firebase_storage/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for Firebase Cloud Storage, a powerful, simple, and
   cost-effective object storage service for Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/firebase_storage
-version: 2.0.1
+version: 2.0.2
```

Comment:
Should be 2.1.0 since we added a new public API. 
https://www.dartlang.org/tools/pub/versioning#semantic-versions

Reply:
good catch, thanks!